dashboard/src/actions/namespace.ts

```diff
@@ -48,6 +48,16 @@ export function fetchNamespaces(
     try {
       const namespaceList = await Namespace.list(cluster);
       const namespaceStrings = namespaceList.namespaces.map((n: IResource) => n.metadata.name);
+      if (namespaceStrings.length === 0) {
+        dispatch(
+          errorNamespaces(
+            cluster,
+            new Error("The current account does not have access to any namespace"),
```

Comment:
just needs a plural for "namespaces" (odd in English, [any is used with plural or uncountable nouns only](https://dictionary.cambridge.org/grammar/british-grammar/any)). Or to use the singular, "does not have access to a namepsace.", but I think "any" is better.